src/icons/OpenLinkIcon.tsx

```diff
@@ -7,13 +7,12 @@ interface Props {
   size?: number
 }
 
-const OpenLinkIcon = ({ size = 12, color = colors.white }: Props) => (
+const OpenLinkIcon = ({ color = colors.white, size = 12 }: Props) => (
```

Comment:
is anything changing here other than args being flipped? if nothing meaningful changes, can we revert?

Reply:
Staying consistent with the order of props.